radicle/src/cob/store.rs

```diff
@@ -137,10 +134,7 @@ where
     ) -> Result<(ObjectId, T, Lamport), Error> {
         let contents = actions
             .into()
-            .iter()
-            .map(encoding::encode)
-            .collect::<Result<Vec<_>, _>>()?
-            .try_into()
+            .try_map(|e| encoding::encode(&e))
             .expect("the collection is always non-empty");
```

Comment:
Is this `expect` needed now? I wouldn't think so. I think it's working on the `Result` type now.

Reply:
Oh right, because the list is NonEmpty to start with!

Reply:
:brain: big brain strats